src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
```suggestion
    BLACK,
```
enum values are written as constants